code/game/gamemodes/clock_cult/clock_scriptures/scripture_drivers.dm

```diff
@@ -98,38 +98,69 @@
 	Click your slab to cancel.</b></span>"
 
 
-//Geis: Grants a short-range binding attack that allows you to mute and drag around a target in a very obvious manner.
-/datum/clockwork_scripture/ranged_ability/geis
-	descname = "Melee Mute & Stun"
-	name = "Geis"
-	desc = "Charges your slab with divine energy, allowing you to bind and pull a struck heretic."
-	invocations = list("Divinity, grant me strength...", "...to bind the heathen!")
+//Abscond: Used to return to Reebe.
+/datum/clockwork_scripture/abscond
+	descname = "Return to Reebe"
+	name = "Abscond"
+	desc = "Yanks you through space, returning you to home base."
+	invocations = list("As we bid farewell, and return to the stars...", "...we shall find our way home.")
 	whispered = TRUE
-	channel_time = 20
-	usage_tip = "You CANNOT TAKE ANY NON-PULL ACTIONS while the target is bound, so Sigils of Submission should be placed before use."
+	channel_time = 50
+	usage_tip = "This can't be used while on Reebe, for obvious reasons."
 	tier = SCRIPTURE_DRIVER
 	primary_component = GEIS_CAPACITOR
 	sort_priority = 5
 	quickbind = TRUE
-	quickbind_desc = "Allows you to bind and mute an adjacent target non-Servant.<br><b>Click your slab to disable.</b>"
-	slab_overlay = "geis"
-	ranged_type = /obj/effect/proc_holder/slab/geis
-	ranged_message = "<span class='sevtug_small'><i>You charge the clockwork slab with divine energy.</i>\n\
-	<b>Left-click a target within melee range to bind!\n\
-	Click your slab to cancel.</b></span>"
-	timeout_time = 100
+	quickbind_desc = "Returns you to Reebe."
+
+/datum/clockwork_scripture/abscond/check_special_requirements()
+	if(invoker.z == ZLEVEL_CITYOFCOGS)
```

Comment:
if you're that worried about spatial gateway, just make it unable to be used on reebe before scripts get unlocked! don't have two scriptures that do nearly the same thing!